.github/labels.yml

```diff
@@ -0,0 +1,109 @@
+# Please add the labels which are applicable to all repos in The Guild Stack here
```

Comment:
@tvvignesh is there a way to put this in a more global place? like a repo under https://github.com/the-guild-org/ ?

Reply:
@dotansimha I was also wondering the same, but sadly not in this action. I noticed that it is possible in the other action though: https://github.com/marketplace/actions/label-syncer#sync-labels-on-another-repository

Can try that and see.

Reply:
@dotansimha  The problem though is that its possible within an org or user. GITHUB TOKEN will not be able to access other repos.